pkg/features/versioned_kube_features.go

```diff
@@ -634,6 +634,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	ServiceAccountTokenJTI: {
 		{Version: version.MustParse("1.29"), Default: false, PreRelease: featuregate.Alpha},
 		{Version: version.MustParse("1.30"), Default: true, PreRelease: featuregate.Beta},
+		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
yeah, auto rebase changed the feature gate lol

Reply:
We recently made a change to add a line break between every feature which is what caused this, I don't think it should be a persistent problem with other PRs.